scripts/080-kch.sh

```diff
@@ -5,3 +5,10 @@ set -eux -o pipefail
 mv /tmp/packer/kchd/{kchd.service,boot_40.service} /usr/lib/systemd/system/
 chmod 644 /usr/lib/systemd/system/{kchd.service,boot_40.service}
 systemctl enable kchd.service boot_40.service
+
+# Make pyenv python entrypoint globally available
+home=/home/$(id -un 1000)
```

Comment:
Suggestion: We already hardcode the "robot" user elsewhere, might as well do it here too

```suggestion
home=/home/robot
```

Or alternatively:


```suggestion
home=~robot
```

Reply:
I'd like to move towards having a single point that sets the username.